namespace java thrift.test

struct myDoubleStruct {
  1: double a
}
